packages/blade/src/components/TopNav/docs/TopNav.stories.tsx

```diff
@@ -271,15 +320,30 @@ const TopNavTemplate: StoryFn<typeof TopNav> = () => {
             setIsSideBarOpen(false);
           }}
         />
-        <Box marginLeft={{ base: '100%', m: '240px', xl: '264px' }} height="calc(100vh - 58px)">
+        <Box
+          marginLeft={{
+            base: '100%',
+            m: makeSize(SIDE_NAV_EXPANDED_L1_WIDTH_MOBILE),
+            xl: makeSize(SIDE_NAV_EXPANDED_L1_WIDTH_DESKTOP),
```

Comment:
These token are not exposed to consumers no? do they need them?

Reply:
Yes they will need them and I've exposed them otherwise they will have to guess these arbitary pixel values.